src/pages/resultsView/fusion/Fusions.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2018 The Hyve B.V.
```

Comment:
Are this file and similar files with the same header, included by this PR or is it just a rebase issue?